src/elements/a-event/index.js

```diff
@@ -1,10 +0,0 @@
-module.exports = {
-  'a-click': require('./a-click'),
-  'a-event': require('./a-event'),
-  'a-hover': require('./a-hover'),
-  'a-load': require('./a-load'),
-  'a-mousedown': require('./a-mousedown'),
```

Comment:
and if we're already supporting primitives, what's the harm in supporting a few nice helpers out of the box (e.g., `<a-click>`, etc.)?